src/components/common/YourCompany/AssociationType.vue

```diff
@@ -189,6 +189,11 @@ export default class AssociationType extends Mixins(CommonMixin, EnumMixin) {
     this.setBusinessInformation({ ...this.getBusinessInformation, associationType: this.selectedAssociationType })
     this.isEditingAssociationType = false
   }
+
+  @Emit('isEditingAssociationType')
+  @Watch('isEditingAssociationType', { immediate: true })
+  private isEditingAssociationTypeChange (isEditing: boolean): void {
```

Comment:
As per requested in 12895

Reply:
OK, good find.

Looks like we weren't emitting this event here, even though the parent component (YourCompany) was expecting it !?!?

Reply:
Yes, my fault